dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/SystemController.java

```diff
@@ -237,6 +237,24 @@ public void getTasksExtendedJson( @PathVariable( "jobType" ) String jobType, Htt
         renderService.toJson( response.getOutputStream(), notifications );
     }
 
+    @RequestMapping( value = "/tasks/extended/{jobType}", method = RequestMethod.GET, produces = { "*/*",
```

Comment:
Was the 2 controller methods meant to be part of this PR?

Reply:
Kind of...
The Frontend should call the extended endpoint, but they are not using versioned endpoints, and as it is now, they have no way to do so.
This is why for me the right way is to separate the 2 versioned method.

Reply:
Yes, but its not related to the expression fixes.. ?

Reply:
Removed